testing/spectest/mainnet/capella/finality/BUILD.bazel

```
@@ -0,0 +1,14 @@
+load("@prysm//tools/go:def.bzl", "go_test")
+
+go_test(
+    name = "go_default_test",
+    size = "medium",
+    timeout = "short",
+    srcs = ["finality_test.go"],
+    data = glob(["*.yaml"]) + [
+        "@consensus_spec_tests_mainnet//:test_data",
+    ],
+    shard_count = 4,
```

is this shard count appropriate? This looks like 1 test only.

```suggestion
    shard_count = 1,
```